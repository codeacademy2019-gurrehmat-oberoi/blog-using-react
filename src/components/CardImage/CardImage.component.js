import React from 'react';
import './CardImage.css';
const CardImage = (props) => {
  return (
      <img src={require('../../Images/' + props.imageName)} className="CardImage" alt = '' />
  );
};

CardImage.defaultProps = {
  imageName: "abstract.png",
};

export default CardImage;