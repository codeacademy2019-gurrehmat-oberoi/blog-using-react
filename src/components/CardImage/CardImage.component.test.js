import React from 'react';
import renderer from 'react-test-renderer';
import CardImage from './CardImage.component';

describe('CardImage', () => {
  it('should render a CardImage', () => {
    const tree = renderer.create(<CardImage />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})