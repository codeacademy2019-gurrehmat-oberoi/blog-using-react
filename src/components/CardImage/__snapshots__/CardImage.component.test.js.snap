// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardImage should render a CardImage 1`] = `
<img
  alt=""
  className="CardImage"
  src="abstract.png"
/>
`;
